internal/rest/avi_obj_pool.go

```diff
@@ -364,7 +364,7 @@ func (rest *RestOperations) AviPoolCacheAdd(rest_op *utils.RestOp, vsKey avicach
 								statusOption.ObjType = lib.MultiClusterIngress
 							}
 							utils.AviLog.Debugf("key: %s Publishing to status queue, options: %v", updateOptions.ServiceMetadata.IngressName, utils.Stringify(statusOption))
-							status.PublishToStatusQueue(updateOptions.ServiceMetadata.IngressName, statusOption)
+							status.PublishToStatusQueue(updateOptions.ServiceMetadata.HostNames[0], statusOption)
```

Comment:
with each ingress, there could be multiple hostname associated. With this logic, are we taking care of updating those.

Reply:
Yes Akshay, the staus is published for every pool object, and each pool is associated with a hostname, so all hostnames are being accounted for. I also ran the code against multi host ingresses and it was working as expected.